1-js/11-async/01-callbacks/article.md

```diff
@@ -280,17 +280,16 @@ function step3(error, script) {
   if (error) {
     handleError(error);
   } else {
-    // ...continue after all scripts are loaded (*)
+    // ...tiếp tục sau tất cả các hành động trên thành công (*)
   }
 };
 ```
 
-See? It does the same, and there's no deep nesting now because we made every action a separate top-level function.
+Như bạn thấy? Nó là như nhau, và không có lồng nhiều cấp nữa bởi vì ta đã tạo mỗi hành động là mỗi hàm khác nhau.
+Nó có hoạt động, nhưng đoạn mã trông thành nhiều phần khác nhau. Nó khó đọc, và có lẽ bạn nhận cần chú ý hơn khi đọc những hàm trên. Điều đó thật bất tiện, nhất là nếu độc giả chưa quen với mã và không biết cách đọc mã. 
```

Comment:
1. Please move this translate to correct where you translated it. Don't remove line-break with the above sentence.

2. và có lẽ bạn nhận => và có lẽ bạn nhận ra rằng

3. độc giả  => người đọc